examples/chain_request_resubmission.py

```diff
@@ -0,0 +1,59 @@
+"""
```

Comment:
Why is this an example? It could very easily be turned into a script taking a chained request and priority and just resubmitting...